.travis.yml

```diff
@@ -48,9 +48,11 @@ install:
     fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$CXX" = "g++" ]; then export CXX="g++-6"; fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$CXX" == "clang++" ]; then export CXX="clang++-3.8"; fi
+  - pip install -r travis/python_reqs.txt
 
 script:
-  - bash .travis.sh
+  - bash travis/check_style.sh
+  - bash travis/travis_main.sh
```

Comment:
You should also change a reference to the old `travis.sh` file in the shippable script (in the root directory)...